### PR TITLE
I have implemented MM1 Preemptive Priority system properties, input o…

### PR DESCRIPTION
**Pull request by Szabin96**
…utput features, unit test cases, table integration test cases and now the system status is "OK".(QSA: #251)

### CODE REVIEW

File: qsa-application-domain/src/main/java/com/unideb/qsa/domain/system/SystemMM1Preemp.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "SystemMM1Preemp{" + "Lambda1=" + Lambda1 + "Lambda2=" + Lambda2 + ", Mu=" + Mu + ", n=" + n + ", t=" + t + ", r=" + r + '}';
+    }
+
+    @AssertTrue(message = "error.smaller.parameter.LambdaMu")
+    private boolean isLambdaSmallerThanMu() {
+        return (Lambda1 + Lambda2) < Mu;
```
[the remark this answers is not in the document]
Comment (ZoltanSzilagyiCse) [reply qsa-application-domain/src/main/java/com/unideb/qsa/domain/system/SystemMM1Preemp.java]: Also, if not, then at least we should display a different error message  like `Lambdas sum should be higher than mu`.

[the remark this answers is not in the document]
Comment (Szabin96) [reply qsa-application-domain/src/main/java/com/unideb/qsa/domain/system/SystemMM1Preemp.java]: I'm on it.

File: qsa-application-implementation/src/main/resources/messages.properties [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
-error.positive.parameter.mu2 = Az <b>mu<sub>2</sub></b> value cannot be <b>negative</b> or <b>0</b>!
-error.positive.parameter.p1 = Az <b>p<sub>2</sub></b> value cannot be <b>negative</b> or <b>0</b>!
-error.positive.parameter.p2 = Az <b>p<sub>2</sub></b> value cannot be <b>negative</b> or <b>0</b>!
-error.positive.parameter.Pi0 = Az <b>µ<sub>0</sub></b> value cannot be <b>negative</b> or <b>0</b>!
-error.positive.parameter.c = <b>c</b> value cannot be <b>negative</b> or <b>0</b>!
-error.smaller.parameter.eSLambda = <b>E[S]</b> value must be smaller then <b>Lambda</b>!
+error.smaller.parameter.LambdaMu = The <b>λ</b> must be smaller then <b>µ</b>!
+error.positive.parameter.Alpha = The <b>α</b> value cannot be <b>negative</b> or <b>0</b>!
+error.positive.parameter.K = The <b>K</b> value cannot be <b>negative</b> or <b>0</b>!
+error.positive.parameter.mu1 = The <b>mu<sub>1</sub></b> value cannot be <b>negative</b> or <b>0</b>!
```
Comment (ZoltanSzilagyiCse) [inline qsa-application-implementation/src/main/resources/messages.properties]: Can you please change the mu to the actual character?

File: qsa-application-implementation/src/main/resources/messages_en.properties [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
-error.positive.parameter.p1 = Az <b>p<sub>2</sub></b> value cannot be <b>negative</b> or <b>0</b>!
-error.positive.parameter.p2 = Az <b>p<sub>2</sub></b> value cannot be <b>negative</b> or <b>0</b>!
-error.positive.parameter.Pi0 = Az <b>�<sub>0</sub></b> value cannot be <b>negative</b> or <b>0</b>!
-error.positive.parameter.c = <b>c</b> value cannot be <b>negative</b> or <b>0</b>!
-error.smaller.parameter.eSLambda = <b>E[S]</b> value must be smaller then <b>Lambda</b>!
+error.smaller.parameter.LambdaMu = The <b>λ</b> must be smaller then <b>µ</b>!
+error.positive.parameter.Alpha = The <b>α</b> value cannot be <b>negative</b> or <b>0</b>!
+error.positive.parameter.K = The <b>K</b> value cannot be <b>negative</b> or <b>0</b>!
+error.positive.parameter.mu1 = The <b>mu<sub>1</sub></b> value cannot be <b>negative</b> or <b>0</b>!
+error.positive.parameter.mu2 = The <b>mu<sub>2</sub></b> value cannot be <b>negative</b> or <b>0</b>!
```
Comment (ZoltanSzilagyiCse) [inline qsa-application-implementation/src/main/resources/messages_en.properties]: Same here.

**Pull request merged**